src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -206,6 +207,12 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv): mixed
             return null;
         }
 
+        if ('shuffle' === $prefix) {
+            \is_array($env) ? shuffle($env) : throw new RuntimeException(sprintf('Env var "%s" cannot be shuffled, expected array, got "%s".', $name, get_debug_type($env)));
```

Comment:
IMHO this is "abusing" ternary, the more "classical"

```php
            if (!\is_array($env)) {
                throw new RuntimeException(sprintf('Env var "%s" cannot be shuffled, expected array, got "%s".', $name, get_debug_type($env)));
            }

            shuffle($env);
```

would be better

Reply:
Subjective

Reply:
Indeed, hence "IMHO" :wink: but I can try "more objective" arguments:
- it's not using the expression result (BTW: shuffle() always returns true [historically could also return false], and if designed today would probably be `: void`; and `throw` wasn't usable as/in an expression before PHP 8.0)
- it's inconsistent with all the rest of the file (even the general codebase)

Dunno if the CS tool could flag it?